Berksfile

```diff
@@ -9,3 +9,9 @@ group :integration do
   cookbook 'git'
   cookbook 'apt'
 end
+
+group :test do
+  cookbook 'chef-services',
+           '>= 4.0.0',
+           git: 'https://github.com/stephenlauck/chef-services.git'
```

Comment:
We should remove this dependency for now, and start creating tests targeted for chef_stack resources. This will resolve the circular dependency issue and allow travis run.